oozie-to-airflow/tests/mappers/test_decision_mapper.py

```diff
@@ -50,14 +48,17 @@ def test_convert_to_text(self):
         # TODO
         # decision mapper does not have the required EL parsing to correctly get
         # parsed, so once that is finished need to redo tests.
-        mapper = decision_mapper.DecisionMapper(
-            oozie_node=self.decision_node, name="test_id", trigger_rule=TriggerRule.DUMMY
-        )
+        mapper = self._get_mapper()
         res = mapper.convert_to_text()
         ast.parse(res)
 
-    # pylint: disable=no-self-use
     def test_required_imports(self):
-        imps = decision_mapper.DecisionMapper.required_imports()
+        mapper = self._get_mapper()
+        imps = mapper.required_imports()
         imp_str = "\n".join(imps)
         ast.parse(imp_str)
+
+    def _get_mapper(self):
```

Comment:
Is this really necessary? Calling one method is changed to calling another method. This only increases the complexity of the code. I prefer readable tests instead of containing less code.

Reply:
Well, this is a standard practice called "method extraction". It's one of the fundamental principles of refactoring and clean code.
I really see no downside to replacing the same repeated 3 lines of code with a method invocation. This not only makes the code more readable, but also conforms to the DRY principle and in case there's a need to change the code in question, we only need to change it once in the extracted method, not everywhere.

I have nothing against readable tests, I like them very much myself. I don't see, however, why both cannot coexist: readable tests AND readable code.

Reply:
I think I agree with both of you. In this case it would be great if the method is named "get_decision_mapper'. Then it would be both DRY and much more readable.